datajoint/autopopulate.py

```diff
@@ -303,6 +304,7 @@ def _populate1(
                         return key, error if return_exception_objects else error_message
                 else:
                     self.connection.commit_transaction()
+                    logger.info("Successful in populating TABLE: {} - KEY: {}".format(self.target.table_name, key))
```

Comment:
```suggestion
                    logger.info(f"Success populating {self.target.table_name}: {key}")
```

Reply:
Do we really want to log so much?

Reply:
This is to facilitate the "notification" mechanism, e.g. having clear and distinct log messages so that it can be parsed easily by some custom loghandler and then fed into some notification scheme (e.g. send emails, slack messages, etc.)

Hence I purposely have the log message being a bit wordy but hopefully distinctive - `Start populating TABLE`, `Error in populating TABLE`, etc. - so that it is easy to parse the log later. 

`Making...` or `Error making...` may just be too generic.